ee/query-service/app/db/metrics.go

```diff
@@ -91,6 +92,26 @@ func (r *ClickhouseReader) GetMetricResultEE(ctx context.Context, query string)
 				metricPoint.Timestamp = v.UnixMilli()
 			case *float64:
 				metricPoint.Value = *v
+			case *uint64:
```

Comment:
I see clickhouse has unit8, 16, 32, 64, and more. What happens when it is not unit64 or uint8? Should we also consider negative values ( I can't think of such practical conditions but we never know what data people might put for traces) and use intX?

Reply:
the current fix is mainly to handle logs and exceptions basic use case of count(). can we re-think about other types and possibly have a wholistic solution soon (in upcoming releases)

Reply:
because there is also string type option for target/result column which we dont cover it.

Reply:
While some queries may use count I think people will start taking full advantage of raw queries and some may have different type. @ankitnayan what do you think?

Reply:
let's ship more use cases in the next releases.  Unless it will be a breaking change, we should defer thoughts on this

Reply:
It's not a breaking change since the existing ones already work with float. Here we are trying to ensure any kind of int type works, not just the `uint8/64`.